plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/nessie/IcebergNessieTableOperations.java

```diff
@@ -95,7 +95,7 @@ protected void refreshFromMetadataLocation(String newLocation)
     @Override
     protected String getRefreshedLocation(boolean invalidateCaches)
     {
-        table = nessieClient.table(toIdentifier(new SchemaTableName(database, tableName)));
+        table = (IcebergTable) nessieClient.fetchContent(toIdentifier(new SchemaTableName(database, tableName)));
```

Comment:
All the Nessie related changes will be in a separate PR along with the dependency upgrade. Ignore these in this PR.